post_installation_configuration/coreos-layering.adoc

```diff
@@ -53,24 +86,54 @@ RUN rpm-ostree override replace https://example.com/myrepo/haproxy-1.0.16-5.el8.
 
 * *{op-system-base} packages*. You can download {op-system-base-full} packages from the link:https://access.redhat.com/downloads/content/479/ver=/rhel---9/9.1/x86_64/packages[Red Hat Customer Portal], such as chrony, firewalld, and iputils.
 +
-.Example Containerfile to apply the firewalld utility
+.Example on-cluster Containerfile to apply the firewalld utility
 [source,yaml]
 ----
-FROM quay.io/openshift-release-dev/ocp-release@sha256...
-ADD configure-firewall-playbook.yml .
-RUN rpm-ostree install firewalld ansible && \
-    ansible-playbook configure-firewall-playbook.yml && \
-    rpm -e ansible && \
+containerfileArch: noarch
+content: |-
+  FROM configs AS final
+  ADD configure-firewall-playbook.yml .
+  RUN rpm-ostree install firewalld ansible && \
+      ansible-playbook configure-firewall-playbook.yml && \
+      rpm -e ansible && \
+      ostree container commit
```

Comment:
In this example the ADD command will take the file from the local dir, right?
@cheesesashimi  What's the behaviour or OCB in this sicuation? does it copy the file from the nodes? since the build pod can be executed in any node, should the file be present in all the nodes?

Maybe we should  not change this example to on-cluster.

Reply:
In this case, the `configure-firewall-playbook.yml` file would need to be present within the build context. It would not be copied from the node or anywhere else. They would have to either retrieve it using curl or git. That might also be undesirable from an on-cluster build perspective because we don't have a way for them to inject secrets to clone from a private repository (this is intentional because we're not trying to replicate a more fully-featured image build mechanism).

Reply:
@cheesesashimi @sergiordlr Should we remove these two examples for on-build, or is there a better solution?

Reply:
@cheesesashimi @sergiordlr I removed the on-cluster and off-cluster firewalld examples and the libreswan example for on-cluster. WDYT? 

Preview: https://75955--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/coreos-layering.html#coreos-layering-examples

Reply:
Perfect. Thank you very much!